pkg/services/dashboards/service/dashboard_service.go

```diff
@@ -982,6 +983,21 @@ func (dr *DashboardServiceImpl) GetDashboardsByPluginID(ctx context.Context, que
 	return dr.dashboardStore.GetDashboardsByPluginID(ctx, query)
 }
 
+// (sometimes) called by the k8s storage engine after creating an object
+func (dr *DashboardServiceImpl) SetDefaultPermissions(ctx context.Context, key *resource.ResourceKey, id claims.AuthInfo, obj utils.GrafanaMetaAccessor) error {
+	ctx, span := tracer.Start(ctx, "dashboards.service.setDefaultPermissions")
+	defer span.End()
+
+	logger := logging.FromContext(ctx)
+
+	logger.Info("TODO!!!!! check default permissions",
```

Comment:
@chaudyg @stephaniehingtgen -- this now gets called

But I am not sure how how to convert, or if we should convert

Reply:
Not sure what you meant by "convert" ? I've implemented the checks for now.

Note that I am currently keeping the existing permissions as-is ... and only checking the permissions when behind the kubernetesDashboard FT.